src/models/user.ts

```diff
@@ -1,17 +1,39 @@
-import { v4 as uuidv4 } from 'uuid'
+import { UserInfoType } from './../types/userType'
+import { v4 as uuid } from 'uuid'
+
+import adapter from '../adapter'
+
+const userAdapter = new adapter('./routes/user', 'user.json')
```

Comment:
`const adapter = new Adapter('./routes/user', 'user.json')`
also you can put it to your class **UserSercvice**

Reply:
Done